src/zarr/buffer/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
+Public API for implementations of the Zarr Buffer interface.
+
+See Also
+========
+arr.abc.buffer: Abstract base class for the Zarr Buffer interface.
+"""
+
+from ..core.buffer import default_buffer_prototype
```

Comment:
```suggestion
from zarr.core.buffer import default_buffer_prototype
```
AFAIK it's a bit safer and clearer to do an absolute import.